src/integration-test/kotlin/integration/helper/IntegrationHelperUtil.kt

```diff
@@ -243,6 +278,37 @@ class IntegrationHelperUtil {
         )
     }
 
+    /**
+     * Transfer asset in iroha with custom creator
+     * @param creator - iroha transaction creator
+     * @param kp - keypair
+     * @param srcAccountId - source account id
+     * @param destAccountId - destination account id
+     * @param assetId - asset id
+     * @param description - transaction description
+     * @param amount - amount
+     * @return hex representation of transaction hash
+     */
+    fun transferAssetIrohaFromClient(
```

Comment:
Could we somehow reuse ```transferAssetIroha``` from ModelUtil? It's quite confusing to have 2 very similar methods in different places.